autoextract/constants.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 
 # Name of the environment variable with the API key
-ENV_VARIABLE = 'SCRAPINGHUB_AUTOEXTRACT_KEY'
+ENV_VARIABLE = 'ZYTE_AUTOEXTRACT_KEY'
 
 # Endpoint
-API_ENDPOINT = 'https://autoextract.scrapinghub.com/v1/extract'
+API_ENDPOINT = 'https://autoextract.zyte.com/v1/extract'
```

Comment:
The new endpoint is not yet alive AFAIK. The release of this repo must wait for it then.

Reply:
Hey! I think there is no reason to delay the release or a merge for the URL change. We could keep the old URL and make a release; it is an internal detail anyways.